admin/menues.rb

```diff
@@ -64,6 +64,11 @@
         link_to(I18n.t("active_admin.menues.form.index.search_link1"), new_admin_article_path(article: {title: menue.title, url_name: menue.target.to_s.split('/').last}), class: "create", title: I18n.t("active_admin.menues.form.index.search_title1"))
       end
     end
+    column I18n.t("active_admin.menues.form.index.column6") do |menue|
+      if menue.parent_id.present?
+        Goldencobra::Menue.find(menue.parent_id).title
```

Comment:
The `menue` could have a `parent_id` but the record for this `id` could be missing. This could lead to a `NoMethodError "title"` for `nil` here.

Reply:
good to know, I will work on that